src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -208,6 +208,32 @@ class QGESSCostingData(FlowsheetCostingBlockData):
             "interest rate.",
         ),
     )
+    CONFIG.declare(
+        "capital_loan_repayment_period",
+        ConfigValue(
+            default=10,
+            domain=float,
+            description="Length of loan repayment period in years.",
+        ),
+    )
+    CONFIG.declare(
+        "debt_percentage_of_CAPEX",
+        ConfigValue(
+            default=50,
+            domain=float,
+            description="Percentage of CAPEX financed by debt; ignored if "
+            "debt_expression is not None. The value should be a percentage, "
+            "for example 10 for a debt corresponding to 10% of the CAPEX. Set "
+            "to zero to indicate no loans are taken out on capital.",
+        ),
+    )
+    CONFIG.declare(
+        "debt_expression",
+        ConfigValue(
+            default=None,
+            description="Set the value or expression to calculate total debt.",
```

Comment:
This would be a Pyomo Expression using flowsheet variables. I'll add a note that the units should be in flowsheet dollars. It's the user's choice, this could be an Expression using any variables - e.g. a percentage of the total CAPEX + a different percentage of the total OPEX, a percentage of the total CAPEX for a subset of equipment accounts, or any other custom expression the user wants to define.